Assets/MRTK/Core/Services/BaseDataProviderAccessCoreSystem.cs

```diff
@@ -152,12 +152,31 @@ public virtual IMixedRealityDataProvider GetDataProvider(string name)
         /// </summary>
         protected bool RegisterDataProvider<T>(
             Type concreteType,
+            string providerName,
             SupportedPlatforms supportedPlatforms = (SupportedPlatforms)(-1),
             params object[] args) where T : IMixedRealityDataProvider
         {
             return RegisterDataProviderInternal<T>(
-                true, // Retry with an added IMixedRealityService parameter
+                true, // Register the provider
```

Comment:
Why the comment change from retry -> register? The parameter name is retryWithRegistrar + the name of this function is register so I would assume that it's trying to register.

Reply:
mistake :)